mypy/semanal.py

```diff
@@ -1461,8 +1461,12 @@ def handle_missing_overload_implementation(self, defn: OverloadedFuncDef) -> Non
                         item.func.abstract_status = IS_ABSTRACT
                     else:
                         item.abstract_status = IS_ABSTRACT
+            elif all(
+                isinstance(item, Decorator) and item.func.abstract_status == IS_ABSTRACT
+                for item in defn.items
+            ):
+                pass
```

Comment:
Is this a behaviour change? If yes, maybe add a test, or would this be better included in a separate PR if mypyc doesn't support this yet?

Reply:
Yeah, I didn't want to add an implementation for couple overloaded methods with many parameters, but unfortunately it didn't work with mypyc. I will put it into a separate PR.